staging/src/k8s.io/apiserver/pkg/authentication/group/token_group_adder_test.go

```diff
@@ -26,10 +27,13 @@ import (
 )
 
 func TestTokenGroupAdder(t *testing.T) {
+	response := &authenticator.Response{User: &user.DefaultInfo{Name: "user", Groups: []string{"original"}}}
```

Comment:
Force the original slice to have extra capacity:

```suggestion
func TestTokenGroupAdder(t *testing.T) {
	groups := make([]string, 0, 100)
	groups = append(groups, "original")
	response := &authenticator.Response{User: &user.DefaultInfo{Name: "user", Groups: groups}}
```